hoodie-cli/src/main/java/com/uber/hoodie/cli/TableFieldType.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2016 Uber Technologies, Inc. (hoodie-dev-group@uber.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.uber.hoodie.cli;
+
+import java.util.Comparator;
+
+/**
+ * Type of fields in the table
+ */
+public enum TableFieldType {
```

Comment:
Could we reuse the standard java classes like - String.class, Long.class etc - directly ?

Reply:
This will avoid having to define indirections here